src/languages/es.ts

```diff
@@ -353,6 +353,14 @@ export default {
         termsOfService: 'Términos de servicio',
         privacy: 'Privacidad',
     },
+    samlSignIn: {
+        welcomeSAMLEnabled: 'Continua iniciando sesión con el inicio de sesión único de su empresa:',
```

Comment:
Let's change the spanish 'welcomeSAMLEnabled' message too to remove "your company's".